### PR TITLE
smbclient => 4.16.0 and tdb => 1.4.6, talloc => 2.3.3, tevent => 0.11.0, ldb => 2.5.0, add perl_json package

### PR DESCRIPTION
**Pull request by saltedcoffii**
Works on x86_64. needs binaries.

```
CREW_TESTING_REPO=https://github.com/saltedcoffii/chromebrew.git CREW_TESTING_BRANCH=smbclient_4.15.3 CREW_TESTING=1 crew update
```

### CODE REVIEW

**Comment by saltedcoffii** [comment]
This build is failing because ./configure is saying tdb isn't installed but chromebrew says it is

**Comment by satmandu** [comment]
building this now:
```
require 'package'

class Tdb < Package
  description 'tdb is a simple database API for sharing structures between parts of Samba'
  homepage 'https://tdb.samba.org/'
  version '1.4.5'
  license 'GPL-3'
  compatibility 'all'
  source_url "https://www.samba.org/ftp/tdb/tdb-#{version}.tar.gz"
  source_sha256 'bcfced884f7031080998b5c4b1c5dce57567055f79417f86dba40dcde99a0e41'

  depends_on 'docbook_xsl'
  depends_on 'libbsd'
  depends_on 'libxslt'

  def self.build
    system "./configure #{CREW_OPTIONS.sub(/--program-suffix.*/, '')}"
    system 'make'
  end

  def self.install
    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
  end
end
```

**Comment by satmandu** [comment]
But what does `config.log` show? That will tell you what is breaking...

**Comment by uberhacker** [comment]
What is the status of this PR?

**Comment by saltedcoffii** [comment]
see https://github.com/skycocker/chromebrew/pull/6607#issuecomment-1026974740

File: packages/smbclient.rb [hunk 1]
```diff
@@ -3,24 +3,11 @@
 class Smbclient < Package
   description 'Tools to access a servers filespace and printers via SMB'
   homepage 'https://www.samba.org'
-  version '4.14.4'
+  version '4.15.3'
```
Comment (uberhacker) [inline packages/smbclient.rb:6]: New version?  https://download.samba.org/pub/samba/stable/samba-4.16.0.tar.gz

File: packages/tdb.rb [hunk 2]
```diff
@@ -3,34 +3,31 @@
 class Tdb < Package
   description 'tdb is a simple database API for sharing structures between parts of Samba'
   homepage 'https://tdb.samba.org/'
-  version '1.4.3'
+  version '1.4.5'
```
Comment (uberhacker) [inline packages/tdb.rb:6]: New version?  https://www.samba.org/ftp/tdb/tdb-1.4.6.tar.gz

**Comment by satmandu** [comment]
FYI I haven't tried building this smbclient binary recently...

**Comment by satmandu** [comment]
Build of 4.16 succeeded.

Adding `  --disable-python` to build script since we don't actually use any python modules from this build.

**Pull request merged**